documentation/api/metrics/v1/index.markdown

```diff
@@ -103,7 +103,7 @@ available via the `/metrics/v1/mbeans` endpoint.
 
 Additionally, we also support the following explicit names:
 
-**Note:** The use of these explicit names is deprecated; please use, e.g., `/v4/commands` instead.
+**Note:** The use of these explicit names is deprecated; please use, e.g., `/pdb/cmd/v1` instead.
 
```

Comment:
not related to this PR, but we should probably consider renaming this file "mbeans.markdown"